app/uk/gov/hmrc/tariffclassificationfrontend/config/AppConfig.scala

```diff
@@ -36,8 +36,9 @@ class AppConfig @Inject()(val runModeConfiguration: Configuration, environment:
   lazy val assetsPrefix: String = loadConfig("assets.url") + loadConfig("assets.version")
   lazy val analyticsToken: String = loadConfig("google-analytics.token")
   lazy val analyticsHost: String = loadConfig("google-analytics.host")
-  lazy val teamEnrolment: String = loadConfig("auth.enrolments.team")
   lazy val managerEnrolment: String = loadConfig("auth.enrolments.manager")
+  lazy val teamEnrolment: String = loadConfig("auth.enrolments.team")
+  lazy val readOnlyEnrolment: String = loadConfig("auth.enrolments.read-only")
```

Comment:
I presume this needs a corresponding app.config change?

Reply:
I've done app-config-prod for you https://github.com/hmrc/app-config-production/pull/3522/files

Reply:
thanks